Azure-Samples/openai-chat-app-quickstart
# Add support for Github models, stop copying to .env

## Purpose

This PR removes the bad practice of copying "azd env get-values" directly into ".env", since the Python extension auto-loads all those variables.
It also adds support for GitHub models. 

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Run locally for each OPENAI_HOST value
* Deploy and check it works in prod
